posts/2025-05-20-25.0.0.5.adoc

```diff
@@ -0,0 +1,169 @@
+// Contact/Reviewer: tonyreigns
+// // // // // // // // 
+[#telemetry]
+== Send Liberty HTTP access logs to OpenTelemetry using MicroProfile Telemetry 2.0
+
+link:https://openliberty.io/docs/latest/microprofile-telemetry.html[MicroProfile Telemetry 2.0] (`mpTelemetry-2.0`) enables OpenTelemetry to collect and export traces, metrics, and logs.
+
+You can now use the `mpTelemetry-2.0` feature to collect link:https://openliberty.io/docs/latest/access-logging.html[Liberty HTTP access logs] to OpenTelemetry. This new log source is in addition to the previously available Open Liberty runtime log sources (messages, traces, ffdcs, audit) and application logs generated by the `java.util.logging` (JUL) API.
+
+To enable the MicroProfile Telemetry 2.0 feature to collect HTTP access logs, enable the feature and configure the `httpEndpoint` and optionally the `httpAccessLogging` configuration elements. Include the new `accessLog` value in the `source` attribute of the `mpTelemtry` configuration element. Also set the `accessLogFields` attribute to `default` or `logFormat` for the `mpTelemetry` configuration element, as shown in the following `server.xml` example:
+
+[source,xml]
+----
+<featureManager>
+    <feature>mpTelemetry-2.0</feature>
+    ...
+</featureManager>
+
+<httpAccessLogging id="accessLogging" logFormat='%h %u %{t}W "%r" %s %b %D %{R}W'/>
+<httpEndpoint id="defaultHttpEndpoint" accessLoggingRef="accessLogging" httpPort="9080" httpsPort="9443"/>
+
+<mpTelemetry accessLogFields="logFormat" source="message,trace,ffdc,accessLog"/>
+----
+   
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+[#bugs]
+== Notable bugs fixed in this release
+
+
+We’ve spent some time fixing bugs. The following sections describe just some of the issues resolved in this release. If you’re interested, here’s the  link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A25005+label%3A%22release+bug%22[full list of bugs fixed in 25.0.0.5].
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/31347[[PH66379\] Not able to config Liberty 25.0.0.3 + Java 21 with FIPS 140-2]
++
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/31247[WLP version 25.0.0.3 start up intermittently fails with AuthCacheImpl NPE]
++
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/31231[Listing the transaction objects in jndi results in NameClassPair with empty names and implementation class names]
++
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/31228[Update SmallRye OpenAPI to 4.0.9]
++
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/31205[Unclear how to correctly configure library element's folder and path]
++
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/31167[Lookup of a subcontext fails for ejblocal: namespace]
++
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/31057["CWMOT5100I" emitted for MicroProfile Telemetry for /health and /metrics even with "OTEL_SDK_DISABLED=true" set]
```

Comment:
This renders as just "CWMOT5100I" for the link text. Do the quotes need to be escaped somehow so the full title shows up?